pyproject.toml

```diff
@@ -0,0 +1,71 @@
+[tool.poetry]
+name = "openapi_client"
```

Comment:
seems you are executing the cli without setting all the important paramaters. for instance `package-name` etc

Reply:
Done